prestoadmin/package.py

```diff
@@ -92,10 +93,14 @@ def rpm_install(rpm_name):
     nodeps = ''
     if env.nodeps:
         nodeps = '--nodeps '
+   
+    filename = "/%s/.bashrc" % env.user
```

Comment:
I don't like changing the user's .bashrc. Try this: http://stackoverflow.com/questions/8313238/best-way-to-add-an-environment-variable-in-fabric